services/app-api/getDetail.test.js

```diff
@@ -150,11 +150,7 @@ describe("component details are returned", () => {
   it("returns empty object if no results", async () => {
     dynamoDb.get.mockResolvedValueOnce({ notAnItem: "something" });
 
-    await expect(getDetails(validEvent))
-      .resolves.toStrictEqual({})
-      .catch((error) => {
-        console.log("caught test error: ", error);
-      });
+    await expect(getDetails(validEvent)).resolves.toStrictEqual({});
```

Comment:
I know we removed the catch so that we could see the test error, but remember, we have those catches because of that "UnhandledPromiseException" deprecation warning that comes up.  Not saying this is one of those places (I kinda started blanketing the catches), but want to remember that so putting it here!